README.rst

```diff
@@ -1,5 +1,17 @@
-.. image:: https://img.shields.io/travis/pypa/twine/master.svg?label=travis-ci
-   :target: https://travis-ci.org/pypa/twine
+.. image:: https://img.shields.io/pypi/v/twine.svg
+    :target: https://pypi.org/project/twine
+
+.. image:: https://img.shields.io/pypi/pyversions/twine.svg
+    :target: https://pypi.org/project/twine
+   
+.. image:: https://readthedocs.org/projects/twine/badge/?version=latest
+    :target: https://twine.readthedocs.io
+
+.. image:: https://travis-ci.org/pypa/twine.svg?branch=master
```

Comment:
We were using shileds for this, why did we stop?

Reply:
Just a small preference for the default "official" versions, in this case. That said, I like the consistency of Shields, and opportunity for customization. Happy to switch.

Reply:
The important part of shields.io is that they don't track users whose browsers will be loading those images. The other services provide no such guarantee (although ReadTheDocs is likely to be the least malicious/invasive of any of them).

As someone who cares incredibly little about the visuals badges and actively despises their invasive nature, shields is a decent privacy compromise.

Reply:
Thanks for the insight; happy to compromise.